addons/binding/org.openhab.binding.tesla/ESH-INF/thing/models.xml

```diff
@@ -0,0 +1,718 @@
+			<channel id="charge" typeId="charge" />
+			<channel id="chargeenablerequest" typeId="chargeenablerequest" />
+			<channel id="chargeenergyadded" typeId="chargeenergyadded" />
+			<channel id="chargelimit" typeId="chargelimit" />
+			<channel id="chargelimitmaximum" typeId="chargelimitmaximum" />
+			<channel id="chargelimitminimum" typeId="chargelimitminimum" />
+			<channel id="chargelimitsocstandard" typeId="chargelimitsocstandard" />
+			<channel id="chargeidealmilesadded" typeId="chargeidealmilesadded" />
+			<channel id="chargeratedmilesadded" typeId="chargeratedmilesadded" />
+			<channel id="chargerate" typeId="chargerate" />
+			<channel id="chargestartingrange" typeId="chargestartingrange" />
+			<channel id="chargestartingsoc" typeId="chargestartingsoc" />
+			<channel id="chargingstate" typeId="chargingstate" />
+			<channel id="chargetomax" typeId="chargetomax" />
+			<channel id="maxcharges" typeId="maxcharges" />
+			<channel id="chargeport" typeId="chargeport" />
+			<channel id="chargercurrent" typeId="chargercurrent" />
+			<channel id="chargerphases" typeId="chargerphases" />
+			<channel id="chargermaxcurrent" typeId="chargermaxcurrent" />
+			<channel id="chargerpower" typeId="chargerpower" />
+			<channel id="chargervoltage" typeId="chargervoltage" />
+<!-- 			<channel id="color" typeId="color" />
+				<channel id="darkrims" typeId="darkrims" />
+				<channel id="name" typeId="name" />
+ -->			<channel id="driverfrontdoor" typeId="driverfrontdoor" />
+			<channel id="doorlock" typeId="doorlock" />
+			<channel id="driverreardoor" typeId="driverreardoor" />
+			<channel id="drivertemp" typeId="drivertemp" />
+			<channel id="elevation" typeId="elevation" />
+			<channel id="estimatedbatteryrange" typeId="estimatedbatteryrange" />
+			<channel id="headingestimation" typeId="headingestimation" />
+			<channel id="estimatedrange" typeId="estimatedrange" />
+<!-- 			<channel id="european" typeId="european" />
+				<channel id="fan" typeId="fan" />
+ 				<channel id="fastcharger" typeId="fastcharger" />
+				<channel id="fastchargertype" typeId="fastchargertype" />
+ -->			<channel id="flashlights" typeId="flashlights" />
+			<channel id="frontdefroster" typeId="frontdefroster" />
+			<channel id="fronttrunk" typeId="fronttrunk" />
+			<channel id="gpstimestamp" typeId="gpstimestamp" />
+<!-- 			<channel id="distanceunits" typeId="distanceunits" />
+			<channel id="temperatureunits" typeId="temperatureunits" />
+			<channel id="chargerateunits" typeId="chargerateunits" />
+			<channel id="24hourtime" typeId="24hourtime" />
+			<channel id="rangedisplay" typeId="rangedisplay" />
+			<channel id="spoiler" typeId="spoiler" /> -->
+			<channel id="heading" typeId="heading" />
+			<channel id="honkhorn" typeId="honkhorn" />
+			<channel id="idealbatteryrange" typeId="idealbatteryrange" />
+			<channel id="insidetemp" typeId="insidetemp" />
+			<channel id="latitude" typeId="latitude" />
```

Comment:
Could you merge latitude+longitude into a "location" channel, using the location item type?